app/src/main/java/com/example/tour_a_versity_v3/CechInfo.kt

```diff
@@ -5,14 +5,14 @@ import android.os.Bundle
 import android.widget.Button
 import androidx.appcompat.app.AppCompatActivity
 
-class TUC_Info : AppCompatActivity() {
+class CechInfo : AppCompatActivity() {
```

Comment:
Changed the name of this class to match the naming conventions as discussed in class and detailed here:
https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules

Reply:
Yes, good call.